go/vt/vttest/environment.go

```diff
@@ -235,21 +245,31 @@ func tmpdir(dataroot string) (dir string, err error) {
 // randomPort gets a random port that is available for a TCP connection.
 // After we generate a random port, we try to establish tcp connections on it and the next 5 values.
```

Comment:
I don't know why we are checking the next 5 values _(6 in total)_, but I'll keep that behaviour